doc/modules/language-guide/examples/grammar.txt

```diff
@@ -299,6 +299,7 @@
 
 <imp> ::= 
     'import' <id>? '='? <text>
+    'import' '{' <list(<pat_field>, ';')> '}' '=' <text>
```

Comment:
why not just <pat> = text?

Reply:
Agreed the syntax here ought to simply be `<pat>` (or `<pat_nullary>`).

Reply:
Doesn't that allow tuple patterns and all the other things (wildcard and literals)?
What would those even mean?
Or should we just delegate those to the type-checker and accept `_` as a NOP import?

Reply:
8d7fb3cae

Reply:
You could insist the pattern is irrefutable, I think, as with shared function args.

Reply:
> Or should we just delegate those to the type-checker (will error out) and accept _ as a NOP import?

Yes. Don't special-case syntax where types are sufficient to handle it.

Reply:
Cool, that's what I do. I have also added tests for it.

Reply:
@crusso the pattern matches arising from imports have this form
``` scheme
  (LetD (ObjP (cons (VarP cons)) (nil (VarP nil))) (VarE file$lib/ListM.mo))
```
but we have little control about the RHS `(VarE file$lib/ListM.mo)`. What we could do is, when in `check_ir`, detect ids like `file$*` and run `check (Ir_utils.is_irrefutable p)` on the LHS pattern. Is this what you mean?

Reply:
@crusso what about 4c94e004f?

Reply:
Okay now I reuse the coverage check from `typing.ml` but make it an error when its is happening in an error. PTAL!

Reply:
JFTR, `<pat>` gives
```
      menhir mo_frontend/parser.{ml,mli} (exit 1)
(cd _build/default && /nix/store/ih98div9nccad7gjvksxwzhyl0jag1m7-ocaml4.12.0-menhir-20211012/bin/menhir --table --inspection -v --strict mo_frontend/parser.mly --base mo_frontend/parser)
Error: one state has shift/reduce conflicts.
Warning: one shift/reduce conflict was arbitrarily resolved.
```